Formula/elan-init.rb

```diff
@@ -0,0 +1,54 @@
+class ElanInit < Formula
+  desc "Lean Theorem Prover installer and version manager"
+  homepage "https://leanprover-community.github.io/"
+  url "https://github.com/leanprover/elan/archive/v1.0.3.tar.gz"
+  sha256 "fb3ddd8915a0694ead0f3a51fdf8b0a5540f983f44eee0e757339244c522b8ee"
+  license "Apache-2.0"
+  head "https://github.com/leanprover/elan.git"
+
+  depends_on "rust" => :build
+
+  # elan-init will run on arm64 Macs, but will fetch Leans that are x86_64.
+  depends_on arch: :x86_64
+
+  def install
+    cargo_home = buildpath/"cargo_home"
+    cargo_home.mkpath
+    ENV["CARGO_HOME"] = cargo_home
```

Comment:
Why do we need this? Note that `brew` sets its own `CARGO_HOME` during the build, and I'm not sure why we're overriding that.

Reply:
Here I again was just trying to copy what I saw other formulas doing, in this case [rustup-init](https://github.com/Homebrew/homebrew-core/blob/master/Formula/rustup-init.rb#L26-L28) which is what I originally was following. But can certainly remove it if you suggest so.

Reply:
Yea I don't know why `rustup-init` does that. I suspect that was added at a time `brew` didn't manage the `CARGO_HOME` env variable.